cluster/addons/e2e-rbac-bindings/random-addon-grabbag.yaml

```diff
@@ -1,20 +0,0 @@
-# TODO remove this
-# currently, the kube-addon-manager is adding lots of pods which all share
-# the system:serviceaccount:kube-system:default identity.  We need to subdivide
-# those service accounts, figure out which ones we're going to make bootstrap roles for
-# and bind those particular roles in the addon yaml itself.  This just gets us started
-apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Bounced this around w/ @roberthbailey and @Q-Lee. I'm starting to lean towards just blowing away this binding, and including instructions in the release note on how to get it back if you really were doing something weird inside your kube-system namespace that needed it.